src/libguac/client.c

```diff
@@ -478,6 +478,29 @@ int guac_client_load_plugin(guac_client* client, const char* protocol) {
 
 }
 
+int guac_client_owner_send_required(guac_client* client, const char** required) {
+
+    /* Don't send require instruction if client does not support it. */
+    if (!guac_client_supports_require(client))
+        return -1;
```

Comment:
Is it `required` or `require`?

Reply:
Should be `required` - I believe I've updated everything correctly, but will take another look through to make sure.

Reply:
Okay, went through and updated the comments in various places as well - this should be consistently documented as `required`.